### PR TITLE
PMM-7660 go modules

### PR DESCRIPTION
**Pull request by Dasio**
https://jira.percona.com/browse/PMM-7660

### CODE REVIEW

**Review by artemgavrilov — COMMENTED** [review]
Build is failed

**Comment by Dasio** [comment]
Build failing shouldn't be related to this PR

**Comment by artemgavrilov** [comment]
@Dasio please fix this `//go:build ignore // +build ignore` problem in this repo

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,8 @@
+BASE_PATH = $(shell pwd)
+BIN_PATH := $(BASE_PATH)/bin
+
+export PATH := $(BIN_PATH):$(PATH)
+
```
Comment (artemgavrilov) [inline Makefile:5]: I'm still not sure that we need this.

**Review by artemgavrilov — DISMISSED** [review]
LGTM

**Review by oter — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Dasio) [reply Makefile:5]: It's already in other repositories.
If we want really get rid of it, let's discuss it again on Go sync. But for now I would like to merge it.

Comment (artemgavrilov) [reply Makefile:5]: Already discussed it with @oter. Yep, there is some benefits from this, so let's keep it.

**Review by artemgavrilov — APPROVED** [review]

**Review by adivinho — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**